src/customerproductorder/Classes/Customer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package customerproductorder.Classes;
```

Comment:
Please read name conventions for Java. Package name is not according to convention.

Use PMD/Checkstyle to catch this problems early.